views/signup.handlebars

```diff
@@ -0,0 +1,29 @@
+    <section class="p-2">
+        <div class="card text-center">
+            <div class="card-header">
+              Create an account
+            </div>
+            <div class="card-body">
+                <form>
+                    <div class="form-group p-2">
+                        <label for="createFirstName">First Name</label>
```

Comment:
`name` attribute needs to be added to respective `<input>` elements: `first_name`, `last_name`, `email`, `password` in order to properly connect to api routes and database